package.json

```diff
@@ -5,11 +5,12 @@
   "main": "index.js",
   "scripts": {
     "clean": "rimraf build",
+    "build:i18n": "i18next 'app/**/*.js' 'server/**/*.js' ",
     "build:server": "babel -d ./build/server ./server && babel -d ./build/shared ./shared && cp package.json ./build && ln -sf \"$(pwd)/webpack.config.dev.js\" ./build",
     "build:webpack": "webpack --config webpack.config.prod.js",
     "build": "yarn clean && yarn build:webpack && yarn build:server",
     "start": "node ./build/server/index.js",
-    "dev": "nodemon --exec \"yarn build:server && node build/server/index.js\" -e js --ignore build/ --ignore app/",
+    "dev": "nodemon --exec \"yarn build:server && cp -R shared/translations/*.json build/shared/translations && node build/server/index.js\" -e js --ignore build/ --ignore app/",
```

Comment:
This copy step should probably happen as a sub-command of `build:server`, as it'll be needed in production too

Reply:
done